custom/icds_reports/filters.py

```diff
@@ -85,7 +85,7 @@ def _get_ancestor_loc_dict(location_list, location):
             if ancestor_loc_dict_in_json is None:
                 break
 
-            child_locations = ancestor.child_locations(include_archive_ancestors=True)
+            child_locations = ancestor.child_locations()
```

Comment:
why this change?

Reply:
This was making it only return archieve locations.

Reply:
i am confused. That says "include archived ancestors", you mean you didn't want archived ancestors and just child locations? if yes, that sounds good

Reply:
offlined and changed it to get_children()